### ISSUE TITLE
How to Limit the TPS(QPS)

### ISSUE THREAD

**Issue by chengaomin** [naver/ngrinder#92]
I used Jmeter before.
Jmeter has the listener —— “Constant Throughput Timer” .
It can control the max Throughput in per minute.
Do you know the nGrinder has the same feature? How to control the max Throughput (TPS)?

Thanks

**Comment by junoyoon** [comment]
No. We don't provide this feature. However it seems very interesting. We will discuss internally we will include this feature in nGrinder 3.5 release.

**Comment by chengaomin** [comment]
really? It sounds quite excited!
How long will I have to wait 3.5?

**Comment by junoyoon** [comment]
@chengaomin  Not decided yet.
We're planning to adopt Spring Boot. so It will take time.

**Comment by GwonGisoo** [comment]
I think it is possible to control in scripting.
How about provide a util method like below:
```
@After
public void after() {
    ExecuteUtil.sleepForMaxTps(100);
}
```

**Comment by junoyoon** [comment]
@GwonGisoo  Great Idea!

**Comment by jen6** [comment]
Is there any update on this?
